contracts/Bondv4.sol

```diff
@@ -225,18 +224,24 @@ contract Bond is iBEP20 {
         }
         return true;
     }
+    function claimForMember(address asset) external returns (bool){
+        uint claimA = calcClaimBondedLP(msg.sender,asset);
+            if(claimA>0){
+               BondVault(bondVault).cFMember(asset,msg.sender);
+            }
+        return true;
+    }
     
     function calcClaimBondedLP(address bondedMember, address asset) public returns (uint){
-        require(isListed[asset], '!listed');
```

Comment:
Remove 'isListed' check in case an asset gets delisted from Bond, allowing normal continued access to claiming the released bond until complete